app/forms/schools/form.rb

```diff
@@ -62,7 +62,11 @@ def school_id
       raise NotImplementedError
     end
 
-    def fields_to_ignore_before_stash_or_save
+    def fields_to_ignore_before_save
```

Comment:
This is to maintain the existing logic in order to prevent any potential issues (i don't know what the implications are for only excluding these fields when just saving so added to both to maintain existing behaviour.